src/cloud-api-adaptor/podvm/Makefile.inc

```diff
@@ -169,7 +170,10 @@ $(GUEST_COMPONENTS_SRC):
 	$(call git_clone_repo_ref,$(GUEST_COMPONENTS_REPO),$(GUEST_COMPONENTS_SRC),$(GUEST_COMPONENTS_VERSION))
 
 $(ATTESTATION_AGENT): $(FORCE_TARGET) | $(GUEST_COMPONENTS_SRC)
-	cd "$(GUEST_COMPONENTS_SRC)/attestation-agent" && CC= ARCH=$(ARCH) $(MAKE) KBC="$(AA_KBC)" ttrpc=true LIBC="$(LIBC)"
+	cd "$(GUEST_COMPONENTS_SRC)/attestation-agent" && CC= ARCH=$(ARCH) $(MAKE) \
+		$(if $(AA_ATTESTER),ATTESTER=$(AA_ATTESTER),) \
```

Comment:
It looks like we are using all-attesters now:
```
cd attestation-agent && RUSTFLAGS=" -C linker=s390x-linux-gnu-gcc" cargo build --release --no-default-features --features "coco_as,kbs bin,ttrpc all-attesters,openssl" --bin ttrpc-aa --target s390x-unknown-linux-gnu
```
and the tss libraries are only present on x86, so the s390x build it failing.

It's also worth noting that the se attester isn't included in `all-attesters` either:
https://github.com/confidential-containers/guest-components/blob/3f2fd793c0a67c74f7ce62b115ed5be293616386/attestation-agent/attestation-agent/Cargo.toml#L61
@huoqifeng - was this done deliberately due to different arch support?

Reply:
The x86 binaries build is also failing with:
```
 The system library `openssl` required by crate `openssl-sys` was not found.
#10 587.3   The file `openssl.pc` needs to be installed and the PKG_CONFIG_PATH environment variable must contain its parent directory.
#10 587.3   The PKG_CONFIG_PATH environment variable is not set.
#10 587.3 
#10 587.3   HINT: if you have installed the library, try setting PKG_CONFIG_PATH to the directory containing `openssl.pc`.
```
so we might need to add another package like `libssl-dev`?

Reply:
I was assuming that we install those on the CI machine, but since the old build defaulted to sample, I suggest we just set  it as default here, too.

Reply:
Cool - hopefully in future (once we sync versions with kata), we can just consume their binaries for some of these cases and let them solve it properly once there (though I was the one that did the guest-components build update, so it might need a closer look!)

Reply:
afaiu the "kata stance" has been to build TEE-agnostic guest-components to minimize binary variance across deployments. It's probably a bit too ambitious, especially across CPU architectures. if you are bundling e.g. tpm libraries, you'll get a dynamic openssl dep for free, similarly for the se attester I think. all this won't work well with static musl libraries.

So, I assume we'll end up with specific binaries after all, but maybe we can get canonical builds from guest-components (like aa-x86_64-gnu, aa-x86_64-musl, aa-s390x-gnu, ...)

Reply:
> It's also worth noting that the se attester isn't included in `all-attesters` either: https://github.com/confidential-containers/guest-components/blob/3f2fd793c0a67c74f7ce62b115ed5be293616386/attestation-agent/attestation-agent/Cargo.toml#L61 @huoqifeng - was this done deliberately due to different arch support?

Yes, because IBMSE attester code can only been run on a s390x instance, while some other attesters can not been compiled on s390x, I'm not sure whether we should leverage the code in kata CI to build artifacts per TEE, I have a PR in-progress to fix it in https://github.com/kata-containers/kata-containers/pull/9817